src/main/java/edu/wpi/grip/ui/Main.java

```diff
@@ -62,5 +65,11 @@ public final void onFatalErrorEvent(FatalErrorEvent error) {
                 }
             }
         });
+
+        if(event.isFatal()) {
+            System.err.println("Original fatal exception");
+            event.getThrowable().printStackTrace();
```

Comment:
If your console doesn't have infinite scroll on and another exception is thrown from anywhere inside of `PlatformImpl.runAndWait(...` (IE sun's code not our code) then it will be lost.